.github/workflows/contribs.yml

```diff
@@ -0,0 +1,23 @@
+name: Contribs
+on:
+  pull_request_target:
+    types: [opened, reopened]
+
+jobs:
+  contribs:
+    runs-on: ubuntu-latest
+    name: Contribs
+    steps:
+      - name: Contribs
+        uses: carlescufi/action-contribs@main
+        with:
+          github-token: ${{ secrets.NCS_GITHUB_TOKEN }}
+          command: 'external'
+          messages: |
+                    Thank you for your contribution!
```

Comment:
@jdstapes what do you think of this text for external contributors?

Reply:
Don't we accept external contributions?